### DISCUSSION TITLE
Add Description Lists

### DISCUSSION THREAD

**Discussion by kobe-ra** [saadeghi/daisyui#2560] [Ideas / feature requests]
Is there a reason description lists aren't in the daisyui? Or they just haven't been added yet.
I'd be happy with something simple like [tailwind ui example](https://tailwindui.com/components/application-ui/data-display/description-lists)

**Comment by saadeghi** [comment]
It's not added yet.  
I need to come with a structure to cover all the different use cases and all the different designs.
